roadmap/target/classes/META-INF/resources/index.html

```diff
@@ -28,7 +28,7 @@
             <div class="collapse navbar-collapse justify-content-center ">
                 <ul class="navbar-nav ms-auto mb-2 landing-nav-center">
                   <li class="nav-item mr-5 pt-4 ">
-                    <a class="nav-link active" aria-current="page" href="#">Resources</a>
+                    <a class="nav-link active" aria-current="page" href="question.html">Resources</a>
```

Comment:
Resources will be the page that Isabel is (hopefully) working on. change back to /# for now until we get the page from her.

Reply:
I added the question.html just for access, I dont have a good way of opening the page but we can change that later

Reply:
Ok. I made the adjustments for the buttons to **properly** work in my edits so just disregard